src/components/pager/pager.spec.js

```diff
@@ -31,22 +32,26 @@ function render(props, renderType = mount) {
 }
 
 describe('Pager', () => {
+  it('renders the Pager without pageSizeSelection', () => {
+    const wrapper = render({
+      totalRecords: 100,
+      pageSize: 10,
+      showPageSizeSelection: false,
+      onPagination: () => true,
+      theme: mintTheme
+    },
+    TestRenderer.create);
+    expect(wrapper).toMatchSnapshot();
+  });
+
   const props = {
-    currentPage: '1',
     totalRecords: 100,
     onPagination: () => true,
-    pageSize: '10',
+    pageSize: 10,
     showPageSizeSelection: true,
     pageSizeSelectionOptions
   };
 
-  it('renders the Pager correctly with the Classic Theme', () => {
-    const wrapper = render(
-      { ...props, onPagination: () => true, theme: classicTheme },
-      TestRenderer.create
-    );
-    expect(wrapper).toMatchSnapshot();
-  });
   it('renders the Pager correctly with the Mint Theme', () => {
     const wrapper = render(
       { ...props, onPagination: () => true, theme: mintTheme },
```

Comment:
instead of the mint theme it could be tested on `baseTheme` as default

Reply:
fixed 4a893a7